jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/actions/MobileActions.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.jdi.light.mobile.actions;
+
+import com.epam.jdi.light.actions.ActionObject;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.*;
+
+import static com.epam.jdi.light.actions.ActionHelper.*;
+
+/**
+ * Created by Roman Iovlev on 20.03.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+@SuppressWarnings("unused")
+@Aspect
+public class MobileActions {
+    @Pointcut("execution(* *(..)) && @annotation(com.epam.jdi.light.common.JDIAction)")
+    protected void jdiPointcut() { }
+
+    private final String className = "com.epam.jdi.light.mobile.actions.MobileActions";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)